pkg/cli/images.go

```diff
@@ -16,14 +16,16 @@ import (
 )
 
 func NewImage(c client.CommandContext) *cobra.Command {
-	cmd := cli.Command(&Image{client: c.ClientFactory}, cobra.Command{
-		Use:     "image [flags] [APP_NAME...]",
+	image := &Image{client: c.ClientFactory}
+	cmd := cli.Command(image, cobra.Command{
+		Use:     "image [flags] [IMAGE_REPO:TAG|IMAGE_ID]",
```

Comment:
The `Use` here was inconsistent with what I see in the code and from the CLI. I made that adjust here and ran `make gen-docs`.